frontend/src/components/header.js

```diff
@@ -7,13 +7,14 @@ import Link from "./Link";
 
 const Header = () => {
   const browseList = [
-    { id: 1, title: "About", route: "/about" },
-    { id: 2, title: "Connect", route: "/connect" },
-    { id: 3, title: "Next Steps", route: "/next-steps" },
-    { id: 4, title: "Events", route: "/events" },
-    { id: 5, title: "Watch", route: "/watch" },
-    { id: 6, title: "Give", route: "/give" },
-    { id: 7, title: "I'm New", route: "/new" },
+    { id: 1, title: "Home", route: "/" },
```

Comment:
We shouldn't be writing these ids in manually, that makes it hard for when you want to add another page in, like you just experienced. 

Later on, where it maps through these lists, the `map` function automatically gives you an index if you want it as you're iterating through the array. Can we use that please? and change for locationsList as well?

Reply:
this is the issue that react opposes using index: https://www.wisdomgeek.com/development/web-development/react/why-should-we-not-use-index-as-key-in-react-lists/#:~:text=To%20start%20with%2C%20array%20indexes,sibling%20components%20and%20inconsistent%20UI.  this is one of many resources that agree about not using indexes

Reply:
Is there a better mechanism aside from hard coding an id? It seems like that's the norm?

Reply:
> this is the issue that react opposes using index: https://www.wisdomgeek.com/development/web-development/react/why-should-we-not-use-index-as-key-in-react-lists/#:~:text=To%20start%20with%2C%20array%20indexes,sibling%20components%20and%20inconsistent%20UI.  this is one of many resources that agree about not using indexes
> 

Would it be better to use the page title as the key?

Reply:
yes. you can use title.  as long as the keys are unique.